go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/aws/aws-sdk-go v1.37.10
 	github.com/go-logr/logr v0.1.0
 	github.com/google/go-cmp v0.3.1
+	github.com/jaypipes/envutil v1.0.0
```

Comment:
LOL, I was mostly only kidding in referring to my amazing envutil library, Vijay :)

Reply:
haha, I did not catch the joke but I think this works fine unless you want me to remove it. : )